src/test/java/com/epam/tc/hw3/pages/IndexPage.java

```diff
@@ -0,0 +1,226 @@
+    private WebElement loginEnter;
+    @FindBy(id = "user-name")
+    private WebElement userName;
+
+    // header elements
+    @FindBy(xpath = "//a[@href='index.html']")
+    private WebElement headerHome;
+    @FindBy(xpath = "//a[@href='contacts.html']")
+    private WebElement headerContact;
+    @FindBy(xpath = "//a[@class='dropdown-toggle' and contains(.,'Service')]")
+    private WebElement headerService;
+    @FindBy(xpath = "//a[@href='metals-colors.html']")
+    private WebElement headerMetalsColors;
+
+    // images
+    @FindBy(xpath = "//span[contains(@class,'icons-benefit')]")
+    private List<WebElement> imagesList;
+
+    // texts
+    @FindBy(xpath = "//span[contains(.,'include')]")
+    private WebElement benifitText1;
+    @FindBy(xpath = "//span[contains(.,'flexible')]")
+    private WebElement benifitText2;
+    @FindBy(xpath = "//span[contains(.,'multi')]")
+    private WebElement benifitText3;
+    @FindBy(xpath = "//span[contains(.,'base')]")
+    private WebElement benifitText4;
+
+    // frame
+    @FindBy(xpath = "//iframe[@id='frame']")
+    private WebElement iframe;
+    @FindBy(xpath = "//input[@id='frame-button']")
+    private WebElement iframeButton;
+
+    // left elements
+    @FindBy(xpath = "//li[@ui='label' and contains(.,'Home')]")
+    private WebElement leftHome;
+    @FindBy(xpath = "//li[@ui='label' and contains(.,'Contact')]")
+    private WebElement leftContact;
+    @FindBy(xpath = "//li[@class='menu-title' and contains(.,'Service')]")
+    private WebElement leftService;
+    @FindBy(xpath = "//li[@ui='label' and contains(.,'Metals')]")
+    private WebElement leftMetalsColors;
+    @FindBy(xpath = "//li[@class='menu-title' and contains(.,'Elements')]")
+    private WebElement leftElementsPacks;
+
+    @FindBy(xpath = "//a[contains(@class, 'dropdown-toggle')]")
+    private WebElement dropdownService;
+    @FindBy(xpath = "//a[@href = 'different-elements.html']")
+    private WebElement diffentElementPageButton;
+
+
+    public IndexPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    public void login(String name, String password) {
+        loginDropdown.click();
+        loginInput.sendKeys(name);
+        passwordInput.sendKeys(password);
+        loginEnter.click();
+    }
+
+    public String getLoginName() {
+        String name = userName.getText();
+        return name;
+    }
+
+    public boolean headerHomeDisplay() {
+        return headerHome.isDisplayed();
+    }
+
+    public boolean headerContactDisplay() {
+        return headerContact.isDisplayed();
+    }
+
+    public boolean headerServiceDisplay() {
+        return headerService.isDisplayed();
+    }
+
+    public boolean headerMetalsColorsDisplay() {
+        return headerMetalsColors.isDisplayed();
+    }
+
+    public boolean headerHomeTextTest() {
+        String headerHomeText = headerHome.getText();
+        return headerHomeText.equals("HOME");
+    }
```

Comment:
Please move test logic and test data out of page objects

Reply:
Moved